discord/commands/count.js

```diff
@@ -102,6 +102,7 @@ let ComposeCount = async (subreddit, CommandStruct, pages) => {
         after = posts[posts.length-1].name
     }
 
+    let UserColorMapping = []
```

Comment:
Oops- I goofed. It should be an object though, just change [] to {} and we're golden